app/controllers/concerns/set_for_end_of_month.rb

```diff
@@ -1,11 +1,14 @@
 module SetForEndOfMonth
```

Comment:
Not totally related to this work, but this module name seems _super_ generic. I'd have a hard time understanding what it's trying to do (and consequetially, I don't think I quite get what `@for_end_of_month` is for).

Any ideas for better names?

Reply:
`for_end_of_month` comes from the drug stock model. It indicates which month the drug stock report is being filed for. It _could_ be just `month` but we'd also want to change it in the model.

Reply:
Ohhh gotcha, this lines up with the model. Sweet.